### ISSUE TITLE
Video downloaded from youtube is corrupted

### ISSUE THREAD

**Issue by raffaem** [ytdl-org/youtube-dl#22971]
## Checklist

- [X] I'm reporting a broken site support
- [X] I've verified that I'm running youtube-dl version **2019.10.29**
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [X] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
PASTE VERBOSE LOG HERE
```
$ youtube-dl https://www.youtube.com/watch?v=oyAUHmnrQ20 --username [MYUSERNAME] --password [MYPASSWORD] -v
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://www.youtube.com/watch?v=oyAUHmnrQ20', '--username', 'PRIVATE', '--password', 'PRIVATE', '-v']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] youtube-dl version 2019.10.29
[debug] Python version 3.7.4 (CPython) - Windows-10-10.0.18362-SP0
[debug] exe versions: none
[debug] Proxy map: {}
[youtube] Downloading login page
[youtube] Looking up account info
[youtube] Logging in
[youtube] Checking cookie
[youtube] oyAUHmnrQ20: Downloading webpage
[youtube] oyAUHmnrQ20: Downloading video info webpage
[youtube] {146} signature length 108, html5 player vflO1GesB
[youtube] {197} signature length 108, html5 player vflO1GesB
[youtube] {226} signature length 108, html5 player vflO1GesB
[youtube] {227} signature length 108, html5 player vflO1GesB
[youtube] {275} signature length 108, html5 player vflO1GesB
[youtube] {359} signature length 104, html5 player vflO1GesB
[youtube] {360} signature length 104, html5 player vflO1GesB
[youtube] {145} signature length 108, html5 player vflO1GesB
[youtube] {196} signature length 104, html5 player vflO1GesB
[youtube] {224} signature length 108, html5 player vflO1GesB
[youtube] {225} signature length 108, html5 player vflO1GesB
[youtube] {274} signature length 108, html5 player vflO1GesB
[youtube] {357} signature length 108, html5 player vflO1GesB
[youtube] {358} signature length 108, html5 player vflO1GesB
[youtube] {144} signature length 108, html5 player vflO1GesB
[youtube] {195} signature length 104, html5 player vflO1GesB
[youtube] {220} signature length 108, html5 player vflO1GesB
[youtube] {221} signature length 108, html5 player vflO1GesB
[youtube] {222} signature length 104, html5 player vflO1GesB
[youtube] {223} signature length 108, html5 player vflO1GesB
[youtube] {273} signature length 108, html5 player vflO1GesB
[youtube] {317} signature length 108, html5 player vflO1GesB
[youtube] {318} signature length 108, html5 player vflO1GesB
[youtube] {143} signature length 108, html5 player vflO1GesB
[youtube] {194} signature length 108, html5 player vflO1GesB
[youtube] {280} signature length 104, html5 player vflO1GesB
[youtube] {142} signature length 108, html5 player vflO1GesB
[youtube] {279} signature length 108, html5 player vflO1GesB
[youtube] {161} signature length 108, html5 player vflO1GesB
[youtube] {148} signature length 108, html5 player vflO1GesB
[youtube] {149} signature length 104, html5 player vflO1GesB
[youtube] {150} signature length 108, html5 player vflO1GesB
[youtube] {350} signature length 108, html5 player vflO1GesB
[youtube] {351} signature length 108, html5 player vflO1GesB
[youtube] {352} signature length 104, html5 player vflO1GesB
[youtube] oyAUHmnrQ20: Downloading MPD manifest
[debug] Default format spec: best/bestvideo+bestaudio
WARNING: You have requested multiple formats but ffmpeg or avconv are not installed. The formats won't be merged.
[debug] Invoking downloader on 'https://r1---sn-b5gg-ca9e.googlevideo.com/videoplayback?expire=1572878182&ei=BuO_XcORGISR1wLqx5S4BA&ip=131.175.147.14&id=o-ADg96Q3I_6P2KuC60u8VboVlOLQMSuyZj3bCcQC_VDxb&itag=227&aitags=142%2C143%2C144%2C145%2C146%2C161%2C194%2C195%2C196%2C197%2C220%2C221%2C222%2C223%2C224%2C225%2C226%2C227%2C273%2C274%2C275%2C279%2C280%2C317%2C318%2C357%2C358%2C359%2C360&source=youtube&requiressl=yes&mm=31%2C29&mn=sn-b5gg-ca9e%2Csn-hpa7znsz&ms=au%2Crdu&mv=m&mvi=0&pl=16&ctier=A&pfa=5&gcr=it&initcwndbps=1525000&hightc=yes&mime=video%2Fmp4&gir=yes&clen=5972851648&dur=8404.686&lmt=1497075440759945&mt=1572856435&fvip=1&keepalive=yes&fexp=23842630&c=WEB&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cctier%2Cpfa%2Cgcr%2Chightc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&lsparams=mm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AHylml4wRQIhAI64mQ4iuyCsu5nnYQ2eQ-J2IbngHB47IsE-6ZkcRvrHAiBYDjaJnK5tHf-4aKShX0lpk5al0LV8pn59jyFursi0iA%3D%3D&sig=ALgxI2wwRgIhAMnd_7c_wcxKP_XwG0k_m2X06ZchYvrgd_ltI42M1Q8SAiEAj-CVwLvKV68JZia19yVWmeeOtWBf6kd7_9W3nJEOXc0=&ratebypass=yes'
[download] Destination: Batman Begins-oyAUHmnrQ20.f227.mp4
[download]   0.1% of 5.56GiB at  1.05MiB/s ETA 01:30:27
ERROR: Interrupted by user

## Description

WRITE DESCRIPTION HERE

The downloaded video is not playable after the initial "Warner Bros" intro. VLC 3.0.8 plays it until the Warner Bros intro ends, then gets stuck on the Warner Bros picture, although the timeline keeps moving forward. THen if you wait or, equivalently, if you skip forward, screens like this are shown:

![vlcsnap-2019-11-04-09h39m18s613](https://user-images.githubusercontent.com/54762742/68108079-11244180-fee7-11e9-828b-6911be69c62e.png)

ffmpeg (below version information) is not able to convert it. Here is ffmpeg version info

```
ffmpeg version git-2019-11-01-53c21c2 Copyright (c) 2000-2019 the FFmpeg developers
  built with gcc 9.2.1 (GCC) 20191010
  configuration: --enable-gpl --enable-version3 --enable-sdl2 --enable-fontconfig --enable-gnutls --enable-iconv --enable-libass --enable-libdav1d --enable-libbluray --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libtheora --enable-libtwolame --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libzimg --enable-lzma --enable-zlib --enable-gmp --enable-libvidstab --enable-libvorbis --enable-libvo-amrwbenc --enable-libmysofa --enable-libspeex --enable-libxvid --enable-libaom --enable-libmfx --enable-ffnvcodec --enable-cuvid --enable-d3d11va --enable-nvenc --enable-nvdec --enable-dxva2 --enable-avisynth --enable-libopenmpt --enable-amf
  libavutil      56. 35.101 / 56. 35.101
  libavcodec     58. 60.100 / 58. 60.100
  libavformat    58. 33.100 / 58. 33.100
  libavdevice    58.  9.100 / 58.  9.100
  libavfilter     7. 66.100 /  7. 66.100
  libswscale      5.  6.100 /  5.  6.100
  libswresample   3.  6.100 /  3.  6.100
  libpostproc    55.  6.100 / 55.  6.100
```

Here is part of the log returned by ffmpeg
```
[aac @ 000002a087525180] Number of bands (35) exceeds limit (28).
Error while decoding stream #0:1: Invalid data found when processing input
[aac @ 000002a087525180] Sample rate index in program config element does not match the sample rate index configured by the container.
[aac @ 000002a087525180] Inconsistent channel configuration.
[aac @ 000002a087525180] get_buffer() failed
Error while decoding stream #0:1: Invalid argument
[h264 @ 000002a087522980] cabac_init_idc 29 overflow
[h264 @ 000002a087522980] decode_slice_header error
[h264 @ 000002a087522980] no frame!
Error while decoding stream #0:0: Invalid data found when processing input
[h264 @ 000002a087522f40] reference count overflow
[h264 @ 000002a087522f40] decode_slice_header error
[h264 @ 000002a087522f40] no frame!
[aac @ 000002a087525180] channel element 3.0 is not allocated
Error while decoding stream #0:1: Invalid data found when processing input
Error while decoding stream #0:0: Invalid data found when processing input
[aac @ 000002a087525180] channel element 3.11 is not allocated
Error while decoding stream #0:1: Invalid data found when processing input
[aac @ 000002a087525180] Reserved bit set.
[aac @ 000002a087525180] Number of bands (5) exceeds limit (3).
Error while decoding stream #0:1: Invalid data found when processing input
[h264 @ 000002a0875106c0] non-existing PPS 1 referenced
[h264 @ 000002a0875106c0] decode_slice_header error
[h264 @ 000002a0875106c0] no frame!
Error while decoding stream #0:0: Invalid data found when processing input
[aac @ 000002a087525180] Reserved bit set.
[aac @ 000002a087525180] Prediction is not allowed in AAC-LC.
Error while decoding stream #0:1: Invalid data found when processing input
[aac @ 000002a087525180] channel element 3.12 is not allocated
Error while decoding stream #0:1: Invalid data found when processing input
[h264 @ 000002a087466ec0] top block unavailable for requested intra mode -1
[h264 @ 000002a087466ec0] error while decoding MB 1 0, bytestream 53361
[h264 @ 000002a087465340] deblocking_filter_idc 8 out of range
[h264 @ 000002a087465340] decode_slice_header error
[h264 @ 000002a087465340] no frame!
Error while decoding stream #0:0: Invalid data found when processing input
[h264 @ 000002a087466ec0] concealing 8160 DC, 8160 AC, 8160 MV errors in i frame
[aac @ 000002a087525180] channel element 2.5 is not allocated
Error while decoding stream #0:1: Invalid data found when processing input
[aac @ 000002a087525180] Reserved bit set.
[aac @ 000002a087525180] Number of bands (7) exceeds limit (1).
Error while decoding stream #0:1: Invalid data found when processing input
Error while decoding stream #0:0: Invalid data found when processing input
frame=  463 fps=6.0 q=0.0 Lsize=    2691kB time=00:06:00.48 bitrate=  61.1kbits/s speed=4.69x
video:2568kB audio:114kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.308330%
Exiting normally, received signal 2.
```

Here is ffmpeg command line

`ffmpeg -i input.mp4 -c:v libvpx-vp9 -b:v 2M output.webm`

In short, the downloaded video looks corrupted

**Comment by raffaem** [comment]
I then think that youtube-dl should report that it will be unable to download the video BEFORE downloading it. I don't think silently downloading a corrupted video is the solution
